manager/integration/tests/common.py

```diff
@@ -4790,9 +4791,11 @@ def wait_deployment_replica_ready(apps_api, deployment_name,
         # deployment is none if deployment is not yet created
         if deployment is not None and \
            deployment.status.ready_replicas == desired_replica_count:
+            ok = True
             break
 
         time.sleep(DEFAULT_DEPLOYMENT_INTERVAL)
+    assert ok
```

Comment:
Nit: When the `assert ok` statement fails, it doesn't output what was `not ok`, suggest adding a brief message behind `assert ok, f"something"`. Also, you can use the else statement in conjunction with a for loop to execute a block of code if the loop completes normally, so we don't need the extra `ok` variable.
```
    for i in range(DEFAULT_DEPLOYMENT_TIMEOUT):
        # do something and break
    else:
        raise TimeoutError(f"Deployment '{deployment_name}' not ready\n"
                           f"Expected replica count: {desired_replica_count}\n"
                           f"Got replica count: {deployment.status.ready_replicas}")
```

Reply:
Will do.  I was just copying the idiom that is used elsewhere in the tests.  This is clearer on failure.